velox/functions/prestosql/window/tests/SimpleAggregatesTest.cpp

```diff
@@ -83,67 +111,27 @@ TEST_P(MultiAggregatesTest, singlePartitionWithSortOrders) {
 }
 
 TEST_P(MultiAggregatesTest, randomInput) {
-  auto vectors = makeFuzzVectors(
-      ROW({"c0", "c1", "c2", "c3"},
-          {BIGINT(), SMALLINT(), INTEGER(), BIGINT()}),
-      10,
-      2);
-  createDuckDbTable(vectors);
-
-  std::vector<std::string> overClauses = {
-      "partition by c0 order by c1, c2, c3",
-      "partition by c1 order by c0, c2, c3",
-      "partition by c0 order by c1 desc, c2, c3",
-      "partition by c1 order by c0 desc, c2, c3",
-      "partition by c0 order by c1",
-      "partition by c0 order by c2",
-      "partition by c0 order by c3",
-      "partition by c1 order by c0 desc",
-      "partition by c0, c1 order by c2, c3",
-      "partition by c0, c1 order by c2",
-      "partition by c0, c1 order by c2 desc",
-      "order by c0, c1, c2, c3",
-      "partition by c0, c1, c2, c3",
-  };
-
-  testWindowFunction(vectors, overClauses);
-}
-
-TEST_P(MultiAggregatesTest, basicRangeFrames) {
-  SimpleAggregatesTest::testWindowFunction(
-      {makeBasicVectors(50)}, kFrameOverClauses, kRangeFrameClauses);
-}
-
-TEST_P(MultiAggregatesTest, basicRangeFramesWithSortOrders) {
-  SimpleAggregatesTest::testWindowFunction(
-      {makeBasicVectors(50)}, kSortOrderBasedOverClauses, kRangeFrameClauses);
-}
-
-TEST_P(MultiAggregatesTest, singlePartitionRangeFrames) {
-  SimpleAggregatesTest::testWindowFunction(
-      {makeSinglePartitionVector(100)}, kFrameOverClauses, kRangeFrameClauses);
-}
-
-TEST_P(MultiAggregatesTest, basicRowFrames) {
-  SimpleAggregatesTest::testWindowFunction(
-      {makeBasicVectors(50)}, kFrameOverClauses, kRowsFrameClauses);
-}
-
-TEST_P(MultiAggregatesTest, singlePartitionRowFrames) {
-  SimpleAggregatesTest::testWindowFunction(
-      {makeSinglePartitionVector(100)}, kFrameOverClauses, kRowsFrameClauses);
+  auto size = 50;
+  std::vector<VectorPtr> vectors;
+  vectors.emplace_back(makeFlatFuzzVector(BIGINT(), size));
+  // These columns are used for k PRECEDING/FOLLOWING frame bounds. So they
+  // should have values >= 1.
+  vectors.emplace_back(
+      makeFlatVector<int64_t>(size, [](auto row) { return row % 5 + 1; }));
+  vectors.emplace_back(
+      makeFlatVector<int64_t>(size, [](auto row) { return row + 1; }));
+  vectors.emplace_back(makeFlatFuzzVector(INTEGER(), size));
+  vectors.emplace_back(makeFlatFuzzVector(BIGINT(), size));
+  auto input = makeRowVector({"c0", "c1", "c2", "c3", "c4"}, vectors);
+
+  createDuckDbTable({input});
+  testWindowFunction({input}, kBasicFiveColumnOverClauses);
```

Comment:
The fiveColumn overClauses are only needed for this test. They can be locally defined as they were in the past. Please remove them from the WindowTestBase.